gnar-style.gemspec

```diff
@@ -1,5 +1,3 @@
-# coding: utf-8
```

Comment:
I removed this line because it caused the following Rubocop warning:

```
Inspecting 11 files
C..........

Offenses:

gnar-style.gemspec:1:1: C: Unnecessary utf-8 encoding comment.
# coding: utf-8
^^^^^^^^^^^^^^^

11 files inspected, 1 offense detected
```

Reply:
Thanks.